nx_parallel/interface.py

```diff
@@ -35,18 +36,31 @@ class ParallelGraph:
 
     __networkx_backend__ = "parallel"
 
-    def __init__(self, graph_object):
-        self.graph_object = graph_object
+    def __init__(self, graph_object=nx.Graph()):
+        if isinstance(
+            graph_object, (nx.Graph, nx.DiGraph, nx.MultiGraph, nx.MultiDiGraph)
+        ):
+            self.graph_object = graph_object
+        else:
+            try:
+                self.graph_object = nx.Graph(graph_object)
+            except Exception as e:
+                raise TypeError(
+                    f"Invalid input type {type(graph_object)} for ParallelGraph."
+                ) from e
```

Comment:
Does NetworkX raise an error here? When would NetworkX not raise an error and yet a TypeError would still be raised?  

Maybe you can get by without the `try/except` clause and just have `self.graph_object = nx.Graph(graph_object)`

Reply:
Yes, I think NetworkX will raise an error so I have removed it. Thanks!